OpenAPI/LearningHub.Nhs.OpenApi.Repositories.Interface/Repositories/Resources/IResourceVersionFlagRepository.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace LearningHub.Nhs.OpenApi.Repositories.Interface.Repositories.Resources
+{
+    using System.Linq;
```

Comment:
needed?

Reply:
The interface file contains an IQueryable method which depends on the linq reference.